packages/cli/commands/project/cloneApp.js

```diff
@@ -101,20 +98,37 @@ exports.handler = async options => {
     const { exportId } = await cloneApp(accountId, appId);
     const { status } = await poll(checkCloneStatus, accountId, exportId);
     if (status === 'SUCCESS') {
-      const absoluteDestPath = path.resolve(getCwd(), location);
-      const zippedProject = await downloadClonedProject(accountId, exportId);
-
-      await extractZipArchive(
-        zippedProject,
-        appName,
-        path.resolve(absoluteDestPath),
-        { includesRootDir: true, hideLogs: true }
-      );
+      // Ensure correct project folder structure exists
+      const baseDestPath = path.resolve(getCwd(), location);
+      const absoluteDestPath = path.resolve(baseDestPath, 'src', 'app');
+      fs.mkdirSync(absoluteDestPath, { recursive: true });
+
+      // Extract zipped app files and place them in correct directory
+      const zippedApp = await downloadClonedProject(accountId, exportId);
+      await extractZipArchive(zippedApp, name, absoluteDestPath, {
+        includesRootDir: true,
+        hideLogs: true,
+      });
+
+      // Create hsproject.json file
+      const configPath = path.join(baseDestPath, 'hsproject.json');
```

Comment:
Nit: I believe we have a constant for `hsproject.json`

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1115/commits/1031c18d71632dead15b6082e22b6366be615501